### PR TITLE
Halfes stunbaton/stunprod stun duration.

### PR DESCRIPTION
**Pull request by ghost**
**About The Pull Request**
Halfes stunbaton/stunprod stun duration.

**Why It's Good For The Game**
Makes stunbatons fair and balanced.

**Changelog**
Stunbaton stun duration from 14 seconds ro 7.5 seconds
Stunprod stun duration from 10 seconds to 5 seconds.

### CODE REVIEW

File: code/game/objects/items/stunbaton.dm [hunk 1]
```diff
@@ -192,7 +192,7 @@
 	w_class = WEIGHT_CLASS_BULKY
 	force = 3
 	throwforce = 5
-	stunforce = 100
+	stunforce = 5
```
Comment (Tlaltecuhtli) [inline code/game/objects/items/stunbaton.dm]: - [ ]

Comment (nemvar) [reply code/game/objects/items/stunbaton.dm]: Based

**Comment by Swiftkor** [comment]
yeah sure just castrate sec why don’t we. can you even cuff with this stun duration?

File: code/game/objects/items/stunbaton.dm (hunk 1 above)
Comment (BeloneX) [reply code/game/objects/items/stunbaton.dm]: oh fuck

**Review by ghost — COMMENTED** [review]
Fixed it.

**Comment by actioninja** [comment]
This ain't it chief.
I want to completely ruin the stun baton, but just halving the duration with no other changes isn't a good idea.
Also, it's 4 seconds to cuff, so you would still be able to cuff with both these.

**Comment by PKPenguin321** [comment]
le thanos meme

**Comment by HomocidlePancakes** [comment]
Oh man. Apparently high-risk/high-reward is OP now.

**Comment by 4dplanner** [comment]
the new enthusiasm for balance is so beautiful

**Comment by MegaEmpirical** [comment]
I can't imhaine this going well. You going to increase the number of charges at least?

**Comment by bgobandit** [comment]
>that lack of a balance tag

**Comment by ATH1909** [comment]
Bruh moment.

**Comment by plapatin** [comment]
baby's first ided pr?

**Comment by BeloneX** [comment]
yeah sorry this is my first PR that i ever made on github,

**Comment by plapatin** [comment]
i understand

**Comment by Daxxed** [comment]
Good first PR
LGTM

**Comment by AdamElTablawy** [comment]
good first PR 

bad idea

stunbaton/stunprod stun duration is fine, really. if someone with a stunbaton wants to kill you they'll just keep stun-beating you, so the duration is irrelevant at that point, so all this would do is make it much harder to cuff someone.

it also requires you to get into melee range.

stun removal is a good thing but stunprods and stunbatons really aren't a balancing issue.

**Comment by Whoneedspacee** [comment]
yes go ahead and encourage me to murder people into crit so i can cuff them because its impossible to chain stun 4 shitters

**Comment by AdamElTablawy** [comment]
'encourage'

yeah right nothing is going to encourage sec officers to murder and beat people on the spot more than they already do

you could give them a gun that shoots rainbows and instantly teleports prisoners into the brig with a baby seal to keep them company and they'd still run in just to beat the seal to death and break his legs

**Comment by wesoda25** [comment]
Its like its trying to be a 4d PR except its not a good idea

**Comment by Akrilla** [comment]
I feel like you might need to expand on the reasoning behind why this balances them... This just feels arbitrary.

**Comment by vpellen** [comment]
I feel like this needs bigger justification than "balance lol"
Why nerf batons/prods, and why nerf them in this way? What is the desired effect, and how does this PR achieve that?

**Comment by PKPenguin321** [comment]
guys hes halving it and calling it balanced it is a thanos meme it is a meme pr what heck

**Comment by 81Denton** [comment]
Reality is often disappointing

**Pull request closed without merging**